vhdbuilder/packer/components.json

```diff
@@ -409,7 +409,8 @@
       "amd64OnlyVersions": [],
       "multiArchVersions": [
         "1.4.0",
-        "1.5.0-rc1"
+        "1.5.0-rc1",
+        "1.5.1"
```

Comment:
looks good to me if 1.5.1 is 'multi-arch'.

Reply:
yes, 1.5.1 is multi-arch.